src/locales/en/fields.yml

```diff
@@ -181,6 +181,19 @@ selfAssessmentPaymentDate:
   legend: Date of payment
   hint: For example 31 3 2024
   content: ""
+  validation:
+    default: "Date of payment must be a real date"
+    required: "Enter your Date of payment"
+    before: "Date of payment must be today or in the past"
+    numeric-day: "Date of payment must include a day between 1 and 31"
+    numeric-month: "Date of payment must include a month between 1 and 12"
+    numeric-year: "Date of payment must include a valid year, like 2023"
+    date-day: "Date of payment must include a day between 1 and 31"
```

Comment:
Should this one actually be "Date of payment must be a real date"? because this should catch invalid dates like 31 February (scenario 1)